src/services/cronJobs/fillSnapshotBalances.ts

```diff
@@ -46,88 +49,122 @@ export async function addFillPowerSnapshotBalanceJobsToQueue() {
   let offset = 0;
   let isFinished = false;
 
-  const groupByBlockNumbers: {
-    [p: number]: {
+  const groupByTimestamp: {
+    [timestamp: number]: {
       userId: number;
       powerSnapshotId: number;
       walletAddress: string;
     }[];
   } = {};
+
   while (!isFinished) {
     const powerBoostings = await getPowerBoostingSnapshotWithoutBalance(
-      100,
+      20,
```

Comment:
Why have we reduced it? it's for fetching values from our internal db.
```suggestion
      100,
```

Reply:
Done